uwblueprint/ccbc
# [Feat] Loading Screen: Edit Review Page

## Notion ticket link

[Loading Screen: Edit Review Page](https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=1e32030d19334bd393d11b583521c926)



## Implementation description
* adding a isLoading state that is set to true if id is passed into the `CreateReview` element
* if isLoading, returns a Spinner component; else it loads the main page components
* spinner only appears if user is editing a review, not when they're creating one



## Steps to test
1. Add a review and check that the spinner does not appear
2. Edit a review and check that spinner shows up first and then disappears when books appears
3. Check that design matches Figma (https://www.figma.com/file/kEa0TbJ5VKAmI97eGefXs6/Magazine-Review---Subscriber-Experience?node-id=464%3A1728)

Demo:


https://user-images.githubusercontent.com/53016529/170890404-2de97e37-af12-4868-9838-e85e20a08bd4.mov




## What should reviewers focus on?
* Is the vertical position okay?


## Checklist
